sklearn/model_selection/_split.py

```diff
@@ -465,6 +531,7 @@ class GroupKFold(_BaseKFold):
         For splitting the data according to explicit domain-specific
         stratification of the dataset.
     """
+
```

Comment:
This change can be reverted. We don't usually follow the one line after docstring convention...